### ISSUE TITLE
Update examples to use http-hash-router

### ISSUE THREAD

**Issue by bcomnes** [Raynos/http-framework#31]
Right now a few examples use [https://github.com/Raynos/routes-router](routes-router) which has a freaky depreciation warning.  Would it be worth someones time to go through an update the examples to use [http-hash-router](https://github.com/Matt-Esch/http-hash-router) or [routes](https://www.npmjs.com/package/routes) instead?

**Comment by yoshuawuyts** [comment]
Hah, yeah not using a deprecated package would be good. Getting this repo to be a bit more lively again would be cool too (': Happy to help out with examples if / when we know where we're lacking.

**Comment by finnp** [comment]
Yeah I still like to point people here as a starting point for building applications. Would be great if we could keep this active.

**Comment by yoshuawuyts** [comment]
I agree, though somehow this project feels so large in scope that I don't feel comfortable contributing to it as it is right now. It'd be rad if the examples could be trimmed down, and the to-do's in the readme could be cleaned up as a start.

**Comment by finnp** [comment]
@yoshuawuyts Yeah I agree. I think it would actually be nicer to add sensible small examples instead of this more complex examples that were refactored from express.
